src/wayland/shell/xdg/handlers/surface/toplevel.rs

```diff
@@ -205,6 +205,7 @@ where
 pub fn send_toplevel_configure(
     resource: &xdg_toplevel::XdgToplevel,
     configure: ToplevelConfigure,
+    send_capabilities: bool,
```

Comment:
Why do we need this bool, can't we just send capability whenever they have changed?

Reply:
This is what the `bool` represents, same as with `send_bounds`. `send_toplevel_configure` is an internal function called from within `XdgToplevel::send_configure`. We could move this calls to `XdgToplevel`, but so far I kept everything directly defined in the `xdg_toplevel` protocol in a single function and only extensions like `xdg_decoration` are called outside of `send_toplevel_configure`.